modules/mauth-proxy/src/main/java/com/mdsol/mauth/proxy/ProxyConfig.java

```diff
@@ -54,8 +59,28 @@ public String getPrivateKey() {
   public UUID getAppUuid() {
     return appUuid;
   }
-  public boolean isV2OnlySignRequests() {
-    return v2OnlySignRequests;
+
+  public List<MAuthVersion> getSignVersions() { return signVersions; }
+
+  public List<MAuthVersion> getSignVersions(String signVersionsStr) {
+    List<MAuthVersion> signVersions = new ArrayList();
+    List<String> versionList = Arrays.asList(signVersionsStr.toUpperCase().split(","));
+    versionList.forEach(e -> {
+      switch (e.toLowerCase()) {
```

Comment:
Can we trim the input as well? it's common that we end up with an accidental space in the front or end and doing this can help avoid debugging sessions where you doubt your sanity :)

Reply:
updated as suggeted